### PR TITLE
Add FreshRSS icon

### LINKED ISSUE #10698: Request: FreshRSS
### Brand Name

FreshRSS

### Website

https://freshrss.org/

### Popularity Metric

[8.2k stars](https://github.com/FreshRSS/FreshRSS) on GitHub, consistent upward star history trend, [actively maintained](https://github.com/FreshRSS/FreshRSS/graphs/code-frequency).

### Official Resources for Icon and Color

SVG: https://github.com/FreshRSS/freshrss.org/blob/main/static/images/icon.svg (also available at https://freshrss.org/images/icon.svg and displayed on the footer).

Color: `#0062BE` (darker shade of blue on the logo).

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by 10xLaCroixDrinker**
FreshRSS simpleicons preview

**Issue:** closes #10698

**Popularity metric:** [8.4K stars](https://github.com/FreshRSS/FreshRSS)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by uncenter** [comment]
I wonder if there is a way we can show the differences in shading of the original icon in a monochrome form. Maybe the lighter blue sections are outlined instead of solid?

**Comment by 10xLaCroixDrinker** [comment]
> Maybe the lighter blue sections are outlined instead of solid?

Done. Updated prview in description

**Review by uncenter — CHANGES_REQUESTED** [review]
Screenshot 2024-04-27 at 20 10 22 (Inkscape)

I'm seeing differences with the circle in the center.

**Comment by 10xLaCroixDrinker** [comment]
> I'm seeing differences with the circle in the center.

Should be all good now 👍🏻

**Review by uncenter — CHANGES_REQUESTED** [review]
Screenshot 2024-04-27 at 21 31 17 (Inkscape)

Still seeing differences with the blue circle on my end.

**Comment by 10xLaCroixDrinker** [comment]
> Still seeing differences with the blue circle on my end.

Okay _now_ it should be all set

**Review by uncenter — CHANGES_REQUESTED** [review]
Screenshot 2024-04-27 at 21 53 50 (Inkscape)

Shoot, still seeing differences! The differences around the center circle are small enough to not matter but the differences around the outer rings needs to be fixed.

**Comment by uncenter** [comment]
This may be an issue with the precision when you are optimizing it FYI. You can try a precision of 3 or 4 to see if that helps.

**Comment by 10xLaCroixDrinker** [comment]
@uncenter I redid the whole thing and I used a precision of 3 when optimizing.

**Review by uncenter — DISMISSED** [review]

File: icons/freshrss.svg
Comment (adamrusted) [inline icons/freshrss.svg:1]: Unfortunately I'm seeing some extra paths on this one, that can be removed in your Vector software by 'adding' the paths using the pathfinder tool.
image

Would you mind having another look, @10xLaCroixDrinker?

Comment (adamrusted) [reply icons/freshrss.svg:1]: Are you still working on this @10xLaCroixDrinker?

**Pull request closed without merging**